elasticsearch.yml

```diff
@@ -0,0 +1,33 @@
+cluster.name: "docker-cluster"
+network.host: 0.0.0.0
+
+######## Start OpenDistro for Elasticsearch Security Configuration ########
+# WARNING: revise all the lines below before you go into production
```

Comment:
What will happen if someone runs this without adjusting the lines? Would it fail or create an incorrectly configured instance?

Reply:
That comment is from the elasticsearch.yml config copied from the OpenDistro documentation and examples. The default file references some placeholder certificates which should not be used in production systems. Some other fields also might have default values not valid for other people's setups, so it could fail to run properly if you don't double-check the config.